components/shared/compositing/lib.rs

```diff
@@ -254,6 +258,17 @@ impl CrossProcessCompositorApi {
         }
     }
 
+    /// Record the candidate of largest-contentful-paint.
+    pub fn send_lcp_candidate(
+        &self,
+        lcp_candidates: LCPCandidates,
+        pipeline_id: WebRenderPipelineId,
+    ) {
+        let _ = self
```

Comment:
We should handle errors. In this case I think we can just log a warning, similar to the other methods.